payjoin/src/receive/v1/exclusive/error.rs

```diff
@@ -38,8 +38,8 @@ impl From<InternalRequestError> for super::ReplyableError {
     fn from(e: InternalRequestError) -> Self { super::ReplyableError::V1(e.into()) }
 }
 
-impl From<RequestError> for JsonReply {
-    fn from(e: RequestError) -> Self {
+impl From<&RequestError> for JsonReply {
```

Comment:
Same comment here

Reply:
Closing this and tracking this issue here: https://github.com/payjoin/rust-payjoin/pull/750#discussion_r2136216905